site/content/docs/main/contributions/tencent-config.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Use Tencent Cloud Object Storage as Velero's storage destination."
+layout: docs
+---
+
+
+You can deploy Velero on Tencent [TKE](https://console.cloud.tencent.com/tke2/) , or even on any other Kubernetes cluster, but anyway you can use Tencent Cloud Object Store as a destination for Velero’s backups.
+
+
+## Prerequisites
+
+- Registered [Tencent Cloud Account](https://cloud.tencent.com/register).
+- Tencent Cloud COS service, referred to as COS, has been launched.
+- A Kubernetes cluster has been created, cluster version v1.10 or later, and the cluster can use DNS and Internet services normally. If you need to create a TKE cluster, Please refer to [Create a cluster](https://cloud.tencent.com/document/product/457/32189).
+
+## Create a Tencent cloud COS bucket
+
+Create an object bucket for Velero to store backups in the Tencent Cloud COS console. About how to create a bucket, please refer to Tencent Cloud COS [Create a bucket](https://cloud.tencent.com/document/product/436/13309) usage instructions.
+
+Set access to the bucket through the object storage console, the bucket needs to be **read** and **written**, so the account is granted data reading,  data writing permissions, About how to configure See [Permission Access Settings](https://cloud.tencent.com/document/product/436/13315.E5.8D.95.E4.B8.AA.E6.8E.88.E6.9D.83).
```

Comment:
"Set access to the bucket through the object storage console, the bucket needs to be **read** and **written**, so the account is granted data reading,  data writing permissions. For how to configure, see the [permission access settings](https://cloud.tencent.com/document/product/436/13315.E5.8D.95.E4.B8.AA.E6.8E.88.E6.9D.83) Tencent user instructions."

Reply:
thank u, I will fixed it.